api/tests/test_views.py

```diff
@@ -1,64 +1,154 @@
+        employee.refresh_from_db()
+
+        # then
+        assert expected_position == employee.position
+
+    @pytest.mark.django_db
+    def test_summary_cost_of_salaries(self):
+        # given
+        salary_1 = 1000
+        salary_2 = 2000
+        salary_3 = 100
+        expected_sum_of_salaries = salary_1 + salary_2 + salary_3
+        EmployeeFactory(salary=salary_1)
+        EmployeeFactory(salary=salary_2)
+        EmployeeFactory(salary=salary_3)
+
+        # when
+        sum_of_salaries = int(CLIENT.get(_url_not_detail(EMPLOYEES, 'summary_cost_of_salaries')).content)
+
+        # then
+        assert expected_sum_of_salaries == sum_of_salaries
+
+
+class TestIncomes(TestCase):
+
+    @pytest.mark.django_db
+    def test_last_incomes(self):
+        # given
+        IncomeFactory.create_batch(10)
+        expected_incomes_number = 5
+
+        # when
+        response = CLIENT.get(
+            _url_not_detail(INCOMES, "last_incomes", _option('number_of_incomes', expected_incomes_number))
+        )
+        incomes_number = len(json.loads(response.content))
+
+        # then
+        assert expected_incomes_number == incomes_number
+
+    @pytest.mark.django_db
+    def test_sum(self):
+        # given
+        sum_1 = 100
+        sum_2 = 350
+        sum_3 = 1050
+        expected_sum_of_sums = sum_1 + sum_2 + sum_3
+        IncomeFactory(date=timezone.now(), sum=sum_1)
+        IncomeFactory(date=timezone.now(), sum=sum_2)
+        IncomeFactory(date=timezone.now(), sum=sum_3)
+
+        # when
+        sum_of_sums = int(CLIENT.get(_url_not_detail(INCOMES, "sum")).content)
+
+        # then
+        assert expected_sum_of_sums == sum_of_sums
+
+
+class TestEvents(TestCase):
+    @pytest.mark.django_db
+    def test_future_events(self):
+        # given
+        expected_number_of_future_events = 3
+        number_of_past_events = 4
+        FutureEventFactory.create_batch(expected_number_of_future_events)
+        PastEventFactory.create_batch(number_of_past_events)
+
+        # when
+        response = CLIENT.get(_url_not_detail(EVENTS, 'future_events'))
+        number_of_future_events = len(json.loads(response.content))
+
+        # then
+        assert expected_number_of_future_events == number_of_future_events
+
+    @pytest.mark.django_db
+    def test_todays_events(self):
+        # given
+        expected_number_of_todays_events = 3
+        number_of_past_events = 4
+        number_of_future_events = 2
+        FutureEventFactory.create_batch(number_of_future_events)
+        PastEventFactory.create_batch(number_of_past_events)
+        TodaysEventsFactory.create_batch(expected_number_of_todays_events)
+
+        # when
+        response = CLIENT.get(_url_not_detail(EVENTS, 'todays_events'))
+        number_of_todays_events_db = len(json.loads(response.content))
+
+        # then
+        assert expected_number_of_todays_events == number_of_todays_events_db
```

Comment:
You could also assert that these are actually the events we want, e.g. by comparing their IDs.

Reply:
ok